CedarJava/src/main/java/com/cedarpolicy/formatter/PolicyFormatter.java

```diff
@@ -0,0 +1,17 @@
+package com.cedarpolicy.formatter;
+
+import com.cedarpolicy.loader.LibraryLoader;
+import com.cedarpolicy.model.exception.InternalException;
+
+public final class PolicyFormatter {
+
+  static {
+    LibraryLoader.loadLibrary();
+  }
+
+  private PolicyFormatter() {
+  }
+
+  public static native String policiesStrToPretty(String policies)
```

Comment:
More nit: I wonder if it makes sense to make it a method of `PolicySet`. In Rust, we cannot do it because the formatter and `PolicySet` are in different packages. But we should be able to do it here.

Reply:
Do you mean to move this method into the `PolicySet` class?

I don't think that is best because this method is strictly for formatting. The output is a String.
This makes it not as directly related to `PolicySet`, so I think it should be belong in this separate class.

Reply:
I'm fine with it as well.